### PR TITLE
init kubernetes python docker build

### PR DESCRIPTION
**Pull request by mrcoggsworth**
## Status
Ready

## Description
This docker build will be used to interact with Kubernetes clusters on any platform. The use cases that fall under SOAR would ideally be used for investigative purposes but could also be utilized for containment actions, Kubernetes audits, maintenance, as well as deployments using the Kubernetes python client library.

### CODE REVIEW

File: docker/kubernetespy/dynamic_version.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+pip freeze | grep demisto-sdk | grep -o -E '[0-9]{1,2}\.[0-9]{1,2}(\.[0-9]+)?'
```
Comment (JudahSchwartz) [inline docker/kubernetespy/dynamic_version.sh:1]: looks to me like this is failing the build. need to change demisto-sdk to kubernetes

Comment (mrcoggsworth) [reply docker/kubernetespy/dynamic_version.sh]: oh geez what an over sight on my end!

**Comment by mrcoggsworth** [comment]
yay circleci build passed!

**Comment by JasBeilin** [comment]
Hi @mrcoggsworth,
Is there any content item you wish to contribute using this image?

**Comment by mrcoggsworth** [comment]
I have a running POC that currently works outside of the demisto DEV server we have running on my local workstation so it should work. I just a need a valid image to use that can be maintained here to keep track of the kubernetes python client library.

**Review by JasBeilin — APPROVED** [review]
Nice :)

**Comment by BEAdi** [comment]
> @JasBeilin I will be using this image to pull into my organizations xsoar environment to connect an EKS cluster to kubernetes using eks tokens in pure python. If it works and doesn't contain any proprietary information I would then take the code to place on the demisto/content repo for others to use

Hi @mrcoggsworth, updating that since you haven't contributed the content item you are referring to, this docker image will be deleted later this week.

**Comment by mrcoggsworth** [comment]
@BEAdi this is being actively pursued by my team at Citi in an internal SOC use case. It's going to take some time to finalize a product to then ship publicly for external use. I ask that this image which could be used for a multitude of reasons to interact with Kubernetes clusters stay live. 

Would at least a working branch on the content project be acceptable to keep it live for now. Enterprise development does not exactly move quick especially for a banking institution. 

Let me know your thoughts?? 

I can say there is active code and it works from a POC standpoint that I can have ported over on a branch in a week or two. 

Thanks for your time.

**Comment by BEAdi** [comment]
Hi @mrcoggsworth,
A working branch in the content repo will be enough. I will follow up in 2 weeks to see that it is indeed open.

**Comment by mrcoggsworth** [comment]
@BEAdi just an update, branch is created and will be pulling out none proprietary code starting tomorrow and this weekend! Have a great Friday/Weekend!

https://github.com/mrcoggsworth/content/tree/kubernetes_incident_responder

**Pull request merged**